osmosis/README.md

```diff
@@ -112,7 +112,7 @@ export default StoreProvider([wrapCounter], Counter);
 
 ## State Persistence with usePersistedState
 
-In order to simplify working with state that needs to be persisted, this library includes a useful utility hook called `usePersistedState`. The persistence for this hook must be configured, and the user can set this up to store key/value pairs with any persistence layer required by using a simple configuration step on app launch.
+In order to simplify working with a state that needs to be persisted, this library includes a useful utility hook called `usePersistedState`. The persistence for this hook must be configured, and the user can set this up to store key/value pairs with any persistence layer required by using a simple configuration step on app launch.
```

Comment:
IMO, I think this was more correct the way it was

Reply:
What if we say, we are basically saving a variable right?

"In order to simplify working with a state variable that needs to be persisted, this library includes a useful utility hook called `usePersistedState`."

Reply:
After talking to Chace reversed these changes with the article.